### DISCUSSION TITLE
Camera View

### DISCUSSION THREAD

**Discussion by mancda** [CommunityToolkit/Maui#1590] [New Feature Discussions]
Are there plans to support a camera view similar to https://github.com/hjam40/Camera.MAUI ?  This project appears to be abandoned and it is in my estimation vital to have for mobile development in MAUI.

Thanks for your consideration

**Comment by bijington** [comment]
> Are there plans to support a camera view similar to https://github.com/hjam40/Camera.MAUI ? This project appears to be abandoned and it is in my estimation vital to have for mobile development in MAUI.
> 
> Thanks for your consideration

There is active work on introducing a CameraView to this repository at: https://github.com/CommunityToolkit/Maui/pull/1387

As for whether it is similar I don't know exactly on that
